sdk/python/runtime/main.go

```diff
@@ -77,17 +81,21 @@ var tplMain string
 // The others were built to be composable and chainable to facilitate the
 // creation of extension modules (custom SDKs that depend on this one).
 type PythonSdk struct {
+    // Directory with the Python SDK source code
 	SDKSourceDir  *Directory
+
+    // List of patterns to allways include when loading Python modules
```

Comment:
nit :scream: - the indentation here should be tab, `go fmt` should fix it.

Reply:
It's always the same with me, because I don't have the IDE do it automatically 😅